src/field_theory/separable.lean

```diff
@@ -447,6 +448,16 @@ begin
   exact_mod_cast (enat.add_one_le_of_lt hq)
 end
 
+lemma squarefree_of_separable {p : polynomial F} (hp : p ≠ 0)  (hsep : separable p) :
```

Comment:
Maybe rename this lemma to `separable.squarefree`. Then we can write `hsep.squarefree` to prove `squarefree p`.

Reply:
Done it.